src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static Task[] tasks;
+    private static int taskCount;
+
+  // prints a string within two horizontal lines, @param is string to be printed
+    public static void printWithLines(String text) {
+        String line = "____________________________________________________________";
+        System.out.println(line);
+        System.out.println(text);
+        System.out.println(line);
+    }
+
+    public static void printHelloMessage() {
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+        String helloMessage = "Hello! I'm Duke\n" + "What can I do for you?";
+
         System.out.println("Hello from\n" + logo);
+
+        printWithLines(helloMessage);
+    }
+
+    public static void printByeMessage() {
+        String byeMessage = "You've terminated Duke. Have a good day!";
+        printWithLines(byeMessage);
+     }
+
+    public static void addTask(String task) {
+
+        if (task.startsWith("todo")) {
```

Comment:
I think can work on the readability of the code by creating a method for each else if statement.